src/motion/features/layout/Animate.tsx

```diff
@@ -109,6 +113,38 @@ class Animate extends React.Component<AnimateProps> {
         origin = originBox || origin
         target = targetBox || target
 
+        /**
+         * If this element has a projecting parent, there's an opportunity to animate
+         * it relatively to that parent rather than relatively to the viewport. This
+         * allows us to add orchestrated animations.
+         */
+        let isRelative = false
+        const projectionParent = visualElement.getProjectionParent()
+
+        if (projectionParent) {
+            let prevParentViewportBox = projectionParent.prevViewportBox
+            let parentLayout = projectionParent.getLayoutState().layout
+
+            if (prevParent) {
+                if (targetBox) {
+                    parentLayout = prevParent.getLayoutState().layout
+                }
```

Comment:
I think a comment explaining what's going on here might be helpful (for me 😅) down the road!